query/src/test/scala/filodb/query/exec/WindowIteratorSpec.scala

```diff
@@ -301,4 +301,32 @@ class WindowIteratorSpec extends RawDataWindowingSpec {
                                                    .asInstanceOf[ChunkedRangeFunction], queryConfig)()
     chunkedWinIt.map(r => (r.getLong(0), r.getDouble(1))).toList.filter(!_._2.isNaN) shouldEqual windowResults
   }
+
+  it("should not return NaN if value is present at time - staleSampleAfterMs") {
+    val samples = Seq(
+      100000L -> 100d,
+      153000L -> 160d,
+      200000L -> 200d
+    )
+    val windowResults = Seq(
+      100000L -> 100d,
+      200000L -> 200d,
+      300000L -> 200d,
+      400000L -> 200d,
+      500000L -> 200d
+    )
+    val rv = timeValueRV(samples)
+
+    val slidingWinIterator = new SlidingWindowIterator(rv.rows, 100000L,
+      100000, 600000L, 0,
```

Comment:
Lets explicitly add a comment that SlidingWindowIterator requires window to be 0 whereas ChunkedWindowIterator requires window to be the staleSample config; 
It would be useful adding comment in both impl and test code.

Reply:
Added comment